### DISCUSSION TITLE
Trouble reading in a raster brick in R environment

### DISCUSSION THREAD

**Discussion by estacy** [microsoft/PlanetaryComputer#129] [Q&A] [answered]
Hello, 

I'm new to the Planetary Computer. I'm using the NASA MODIS Flood data product (not available on the Hub). I've downloaded hundreds of geotiffs and read them into R on my local machine, and exported the resulting brick as a .grd file. I'm now turning to the Planetary Computer to try to further clean and analyze these rasters. I've uploaded the .grd file into my home directory in the R environment, and I'm running the same code that I use on my local machine but it is not working. On my local machine, this code runs fine and successfully loads in the raster brick. My local machine is using the newest version of the raster package (3.6.3) while the R environment on the Planetary Computer is using the latest package available on conda-forge (3.5_21). I am not familiar with conda-forge but I checked to see if it was possible to update the package on conda-forge and it seems that there are some comments on the request, perhaps some other dependency conflicts that are preventing this update from happening. 

Code: 
`if (!require("pacman")) install.packages("pacman")`
`pacman::p_load(devtools, raster, rgdal, remotes, terra, BiocManager, rasterVis,`
` gdalUtils, sf, exactextractr, tidyverse, parallel, foreach, doParallel)`
`numCores <- detectCores() `
`registerDoParallel(numCores)  # use multicore for parallel computing, set to the number of our cores`
`brk <- stack("raw_brick.grd")`

Error output: 
Error in .rasterObjectFromFile(x, objecttype = "RasterBrick", ...): Cannot create a RasterLayer object from this file.
Traceback:

1. stack("raw_brick.grd")
2. stack("raw_brick.grd")
3. .local(x, ...)
4. .quickStackOneFile(x, bands = bands, native = native)
5. brick(filename, native = native)
6. brick(filename, native = native)
7. .rasterObjectFromFile(x, objecttype = "RasterBrick", ...)
8. stop("Cannot create a RasterLayer object from this file.")

**Comment by jessjaco** [comment, marked as answer]
Did you upload the .gri file too?

**Reply by estacy** [reply]
Ah, thank you! I'm new to geospatial and didn't realize both files needed to be present.
